internal_transfers/actions/tests/e2e/pipeline.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { TenderlySimulator } from "../../src/simulate/tenderly";
+import { internalizedTokenImbalance } from "../../src/pipeline";
+import { getDB } from "../../src/database";
+import { getTxData } from "./helper";
+
+const { TENDERLY_USER, TENDERLY_PROJECT, TENDERLY_ACCESS_KEY } = process.env;
+
+const simulator = new TenderlySimulator(
+  TENDERLY_USER || "INVALID_USER",
+  TENDERLY_PROJECT || "TENDERLY_PROJECT",
+  TENDERLY_ACCESS_KEY || "TENDERLY_ACCESS_KEY"
+);
+
+const db = getDB("postgresql://postgres:postgres@localhost:5432/postgres");
```

Comment:
Nitpick: use a const/env var for the db URL

Reply:
but this is just a test...

Reply:
Fixed [003b4a1](https://github.com/cowprotocol/solver-rewards/pull/261/commits/003b4a18ef0dca10c2b20a717998a0ac86c7216c)